src/test/java/org/openrewrite/java/migrate/jakarta/DeprecatedCDIAPIsRemoved40Test.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.migrate.jakarta;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.InMemoryExecutionContext;
+import org.openrewrite.config.Environment;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+
+public class DeprecatedCDIAPIsRemoved40Test implements RewriteTest {
```

Comment:
I've seen this pattern on a few PRs before; could you update your defaults not to make the test classes or methods public?
```suggestion

class DeprecatedCDIAPIsRemoved40Test implements RewriteTest {
```

Reply:
done.. I will remember that ..I thought they needed to be public based on some earlier Test code I saw